### PR TITLE
luci-mod-system: add config option to invert heartbeat trigger

### LINKED ISSUE #5911: luci-led-trigger: Allow hartbeat to be inverted
## Steps to reproduce:
1. go to: System → LED configuration → Add LED action  → Pick heartbeat as trigger

## Actual behavior:
1. Unlike the custom timer, the hearbeat trigger does not pop out additional options, such as (the only one) a checkbox to `invert`

## Expected behavior:
```
When selecting the heartbeat trigger, I wish to be able to set the `invert` flag.
```

## Additional Information:
I've created https://github.com/openwrt/openwrt/pull/10358 to add support in openwrt itself for this.

OpenWrt version information from system `/etc/openwrt_release`

```
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='SNAPSHOT'
DISTRIB_REVISION='r20152+61-a3e4949998'
DISTRIB_TARGET='realtek/rtl930x'
DISTRIB_ARCH='mips_24kc'
DISTRIB_DESCRIPTION='OpenWrt SNAPSHOT r20152+61-a3e4949998'
DISTRIB_TAINTS='no-all busybox'
```

### PR DESCRIPTION
**Pull request by feckert**
Depends on https://github.com/openwrt/openwrt/pull/10358
Fixes #5911

### CODE REVIEW

File: modules/luci-mod-system/htdocs/luci-static/resources/view/system/led-trigger/heartbeat.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 	trigger: _('Heartbeat interval (kernel: heartbeat)'),
 	description: _('The LED flashes to simulate actual heart beat.') +
 		_('The frequency is in direct proportion to 1-minute average CPU load.'),
 	kernel: true,
-	addFormOptions: function(s) {}
+	addFormOptions: function(s) {
+		var o;
+
+		o = s.option(form.Flag, 'inverted', _('Invert blinking'),
+			_('The LED is more on then off'));
```
Comment (jow-) [inline modules/luci-mod-system/htdocs/luci-static/resources/view/system/led-trigger/heartbeat.js:14]: I would suggest the following phrase:
`When enabled, the LED is lit continuously and flickering on activity. When disabled, the LED is off by default and only flashes on activity.`

**Comment by oliv3r** [comment]
With `inverted` as flag, compared to `invert` that you had in your first branch (which I initially used :p) it obviously didn't work; however naturally, with the proper flag, it works perfectly. So we just need to get https://github.com/openwrt/openwrt/pull/10358 merged. However, it needs still 6 reviewers :S

**Comment by oliv3r** [comment]
is now merged; so this can go in too :)

**Comment by feckert** [comment]
@oliv3r done

**Pull request merged**